apps/wallet/src/ui/app/shared/image-icon/index.tsx

```diff
@@ -10,17 +10,22 @@ const imageStyle = cva(['text-white capitalize overflow-hidden bg-gray-40  shrin
 			sm: 'w-6 h-6 font-medium text-subtitleSmallExtra',
 			md: 'w-7.5 h-7.5 font-medium text-body',
 			lg: 'w-10 h-10 font-medium text-heading4',
-			xl: 'w-15 h-15 font-medium text-heading4',
+			xl: 'w-12.5 h-12.5 font-medium text-heading4',
+			xxl: 'w-15 h-15 font-medium text-heading4',
```

Comment:
Does this xxl variant actually get used?

Reply:
Yea, I just updated it on subsequent pushes